server/WebAPI/DTO/MappingProfile.cs

```diff
@@ -0,0 +1,15 @@
+﻿using AutoMapper;
+using WebAPI.DTO.Trip;
+
+namespace WebAPI.DTO
+{
+  public class MappingProfile : Profile
+  {
+    public MappingProfile()
+    {
+      CreateMap<Entities.Models.Country, CountryResponse>();
```

Comment:
For me, "Response" has more common meaning. Today we send to front only DTO, next day we will want to add pagination for example. And we will need model of Response like 
IResponse {
data;
pagesCount,
currentPage
}
or other things wich doesn`t correlate with these DTO models at all. DTO is more specific and clear name. So, I can suggest to name DTO entities like CountryDTO

Reply:
Ok, renamed